src/supertux/savegame.cpp

```diff
@@ -323,6 +323,13 @@ Savegame::get_worldmaps()
 WorldmapState
 Savegame::get_worldmap_state(const std::string& name)
 {
+  std::string path;
+  if (name[0] == '/') {
+    path = name;
+  }
+  else {
+    path = "/" + name;
+  }
```

Comment:
This check seems needless - it should be on the caller to ensure that there is a leading slash if a leading slash is necessary. (Also, it would be preferable to use the FileUtils where possible.)